Tasks/PackerBuildV1/src/packerHost.ts

```diff
@@ -101,9 +101,10 @@ export default class PackerHost implements definitions.IPackerHost {
         var installedPackerPath = tl.which("packer", false);
         var installedPackerVersion = this._getPackerVersion(installedPackerPath);
         console.log(tl.loc("InstalledPackerVersion", installedPackerVersion));
-        
-        var packerVersionString = this._taskParameters.packerVersionString ? this._taskParameters.packerVersionString : constants.CurrentSupportedPackerVersionString;
-        
+        var packerVersionString = constants.CurrentSupportedPackerVersionString;
+        if(this._taskParameters.templateType == constants.TemplateTypeCustom) {
+            packerVersionString = this._taskParameters.packerVersionString ? this._taskParameters.packerVersionString : constants.CurrentSupportedPackerVersionString;
+        }
         if(!installedPackerVersion || 
             utils.isGreaterVersion(utils.PackerVersion.convertFromString(packerVersionString), utils.PackerVersion.convertFromString(installedPackerVersion))) {
```

Comment:
Lets not use the greater version when an explict version is specified

Reply:
done